ydb/core/fq/libs/ydb/ydb.h

```diff
@@ -77,7 +98,7 @@ struct TGenerationContext : public TThrRefBase {
     EOperationType OperationType = Register;
 
     // within this session we execute transaction
-    NYdb::NTable::TSession Session;
+    ISession::TPtr Session = nullptr;
```

Comment:
Тут кажется nullptr лишний 


```suggestion
    ISession::TPtr Session;
```